crates/language/src/target_language.rs

```diff
@@ -651,6 +673,7 @@ impl TargetLanguage {
             | TargetLanguage::Rust(_)
             | TargetLanguage::Solidity(_)
             | TargetLanguage::Tsx(_)
+            | TargetLanguage::Php(_)
             | TargetLanguage::TypeScript(_) => format!("// {}\n", text),
```

Comment:
This should probably be updated to use the `comment_prefix` method from the Language trait instead of redefining it here.